snapcraft/tests/commands/test_update.py

```diff
@@ -20,19 +20,44 @@
 import yaml
 from testtools.matchers import Contains, Equals, FileExists
 from xdg import BaseDirectory
+from textwrap import dedent
+from unittest import mock
+from unittest.mock import call
 
 from snapcraft.tests import TestWithFakeRemoteParts
+from snapcraft.tests import fixture_setup
 from . import CommandBaseTestCase
 
 
 class UpdateCommandTestCase(CommandBaseTestCase, TestWithFakeRemoteParts):
 
+    yaml_template = dedent("""\
+        name: snap-test
+        version: 1.0
+        summary: test snapping
+        description: if snap is succesful a snap package will be available
+        architectures: ['amd64']
+        type: {}
```

Comment:
When using an external template like this, named parameters are far more clear and scalable than positional ones.

Reply:
I just copied it from the snap test :-D but yeah, I changed it to {snap_type}